### PR TITLE
Update JAVA_OPTS and docker versions

### PR DESCRIPTION
**Pull request by shambupujar**

### CODE REVIEW

File: node-runner-cli/setup/SystemD.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def backup_file(filepath, filename, backup_time):
     def set_environment_variables(keystore_password, node_secrets_dir):
         command = f"""
         cat > {node_secrets_dir}/environment << EOF
-        JAVA_OPTS="-server -Xms8g -Xmx8g -XX:+HeapDumpOnOutOfMemoryError -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector"
+        JAVA_OPTS="--enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m -XX:+HeapDumpOnOutOfMemoryError -XX:+UseCompressedOops -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector"
```
Comment (dhedey) [inline node-runner-cli/setup/SystemD.py:87]: Were all these changes intended? What is the intended purpose of `-XX:+UseCompressedOops` and `-XX:MaxDirectMemorySize=2048m`?

**Review by dhedey — APPROVED** [review]
Looks good.

However - as a heads up, we've already merged some Co-ordinated forks changes into Main. We probably also want to back-port these changes on top of the 1.1.0 in a separate branch in order to create a 1.1.1 node runner cli build.

File: node-runner-cli/setup/SystemD.py (hunk 1 above)
Comment (shambupujar) [reply node-runner-cli/setup/SystemD.py:87]: Both of these changes were missing from previous release. 

 -XX:MaxDirectMemorySize=2048m is used to set the size of the memory that is outside JVM for the process to use. Issue with missed proposals and memory leak was fixed, but this configuration is nice to have to limit the usage of the memory outside JVM 
  -XX:+UseCompressedOops is to reduce jVM memory usage. This setting was tested again during memory leak investigation on missed proposals.  It was missing on CLI as well

**Comment by shambupujar** [comment]
This was cherry picked into release branch. Closing the PR

**Pull request closed without merging**